gensrc/thrift/PlanNodes.thrift

```diff
@@ -749,6 +750,7 @@ struct THdfsScanNode {
     4: optional Types.TTupleId min_max_tuple_id
 
     // The conjuncts that are eligible for dictionary filtering.
+    // note: unused now!
```

Comment:
If it's deprecated, name it as `DEPRECATED_dictionary_filter_conjuncts`

Reply:
fixed.